src/main/resources/application-dev.yml

```diff
@@ -18,8 +18,9 @@ consentmanager:
     url: http://localhost:8080
     jwkUrl: http://localhost:8080/certs
   gatewayservice:
-    baseUrl: http://host.docker.internal:8081/v1
+    baseUrl: http://localhost:8004/v1
     requestTimeout: 1000
+    enabled: true
   userservice:
     url: http://localhost:9000
     transactionPinDigitSize: 4
```

Comment:
u can remove this file

Reply:
we have added one env. var

Reply:
sorry.. I was trying to edit the comment.. you can remove this change baseUrl: http://localhost:8004/v1

Reply:
removed